features/step_definitions/comment_steps.rb

```diff
@@ -19,6 +19,40 @@
   FactoryBot.create(:comment, :by_guest, commentable: work.first_chapter)
 end
 
+ParameterType(
+  name: "commentable",
+  regexp: /the (work|admin post|tag) "([^"]*?)"/,
```

Comment:
Can you use `"(.*?)"` in the regex?

Reply:
I'd really prefer not to, because it would allow this to match to `the work "Hello" by "creator"`. But I can drop the `?` if you think that'd be better.

Reply:
Yeah, sounds good - since the lazy quantifier is not needed in this case.